implementations/c/ockam/log/log/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# ---
+# ockam::log
+# ---
+add_library(ockam_log)
+add_library(ockam::log ALIAS ockam_log)
```

Comment:
@mrinalwadhwa shouldn't it be add_library(ockam::ockam_log ALIAS ockam_log) ?

Reply:
What would be a reason to prefer `ockam::ockam_log` instead of `ockam::log`?

Reply:
I read through pages 145 and 146 of Professional Cmake again, my take away from that is -
* that the external ALIAS should be called `ockam::log`
* we could maybe get away with the internal target being called `log` instead of `ockam_log` but I think `ockam_log` is ok.

Reply:
@mrinalwadhwa it looks like we understand book differently, let's recheck how it works after we start working with installing/packaging the library

Reply:
👍